File: SampleSetup/Protocol/Body/ApiBodyWithNestedRef.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public class ApiBodyWithNestedRef
    {
        public List<ApiBodyWithNested.Nested_1.Nested_1_2> PropNestedRefs { get; set; } = null;
    }
}

File: SampleSetup/Output/PHP_8/Body/ApiBodyWithNestedRef.php
<?php
namespace SampleProject\Protocol\Body;

class ApiBodyWithNestedRef {
    /**
     * @var ApiBodyWithNested.Nested_1.Nested_1_2[]|null
     */
    public $PropNestedRefs;

    public function __construct()
    {
        $this->PropNestedRefs = null;
    }
}
